Customization/T250/CodeSnippets/Step1.5.2/RSSVPaymentPlanInq.cs

```diff
@@ -123,7 +123,7 @@ protected virtual void viewOrder()
                 var graph = PXGraph.CreateInstance<SOOrderEntry>();
                 // set the current property of the graph
                 graph.Document.Current = graph.Document.
-                  Search<RSSVWorkOrder.orderNbr>(order.OrderNbr);
+                  Search<SOOrder.orderNbr>(order.OrderNbr);
```

Comment:
SOOrder should be used instead of RSSVWorkOrder here, because we are querying the view of the Sales Order screen. The reason why this works in current form is the fact that both the SOOrder and RSSVWorkOrder have a field name OrderNbr.